### DISCUSSION TITLE
Frigate on a separate Debian Docker

### DISCUSSION THREAD

**Discussion by jlaitinen81** [blakeblackshear/frigate#760] [Ask A Question] [answered]
Hi all!

I just installed the latest Frigate version on a dedicated NUC with Debian 10 and Docker. The reason for this was to reduce my CPU load on my primary Home Assistant system.

The installation seemed to go fine and I can access the Frigate UI via http://<MY_FRIGATE_IP>:5000 and can see that all my cameras work well.

On the primary HA system I removed the Frigate AddOn, installed the HACS Frigate 1.0.3 Integration and via the HA Config added the Frigate Integration with http://<MY_FRIGATE_IP>:5000. This created the HA sensors and entities, but only some of them update (for example camera fps). Others still show values from the time when I ran Frigate as an AddOn on the same system.

I also can not turn the switches (like switch.reolink_detect) to "on" position, which means that I can not turn detection on.

This leads me to think that there is some link broken between these two systems. I set the mqtt address in the Docker Frigate to point to the Mosquitto MQTT broker on my HA. Was this the right way?

How does the Frigate integration on HA talk to the actual Frigate on a separate machine? Via web api as there is no mqtt broker on that instance...?

Any help to finalize this dual setup would be greatly appreciated...

**Comment by blakeblackshear** [comment, marked as answer]
Make sure frigate can connect to your mqtt server.

**Reply by jlaitinen81** [reply]
I defined the ip of my mqtt server in Frigate's config.yml and I know the server works as it serves many other applications without issues.

How do I make sure Frigate can connect to the server?

**Reply by jlaitinen81** [reply]
Don't know what I did, but after a Frigate restart everything works beautifully 😊.

You rock!
